_internals/configuration.md

```diff
@@ -193,391 +39,321 @@ Sets the first day of the week to a number from 1 through 7.
+This parameter instructs the server to dump the query graph parsed by ANTLR parser to local disk. The path used is
 `/tmp/antlr.dot`.
 
 
 ### `babelfishpg_tsql.enable_antlr_detailed_log`
 
-Enable detailed ATNLR parser logging.
-
-
-### `babelfishpg_tsql.enable_ownership_structure`
-
-Enable Babelfish ownership structure.  This flag is for Babelfish development
-and should not be used by end users.
-
-
-### `babelfishpg_tsql.escape_hatch_compatibility_level`
-
-Suppresses the error when trying to set the compatibility level.
-
-
-### `babelfishpg_tsql.escape_hatch_constraint_name_for_default`
-
-Escape hatch for `DEFAULT` option in `ALTER TABLE ADD` constraint.
-
-
-### `babelfishpg_tsql.escape_hatch_database_misc_options`
-
-Escape hatch for misc options in `CREATE/ALTER DATABASE`.
-
-
-### `babelfishpg_tsql.escape_hatch_for_replication`
-
-Escape hatch for `[NOT] FOR REPLICATION` option.
-
-
-### `babelfishpg_tsql.escape_hatch_fulltext`
-
-Escape hatch for fulltext search.
-
-
-### `babelfishpg_tsql.escape_hatch_index_clustering`
-
-Escape hatch for `CLUSTERED` option in `CREATE INDEX` and constraints.
-
-
-### `babelfishpg_tsql.escape_hatch_index_columnstore`
-
-Escape hatch for `COLUMNSTORE` option in `CREATE INDEX`.
-
-
-### `babelfishpg_tsql.escape_hatch_join_hints`
-
-Escape hatch for join hints.
-
+This parameter enables detailed ANTLR parser logging.
 
-### `babelfishpg_tsql.escape_hatch_language_non_english`
-
-Escape hatch for non-english language.
-
-
-### `babelfishpg_tsql.escape_hatch_login_hashed_password`
-
-Escape hatch for login hashed passwords.
-
-
-### `babelfishpg_tsql.escape_hatch_login_misc_options`
-
-Escape hatch for login miscellaneous options.
-
-
-### `babelfishpg_tsql.escape_hatch_login_old_password`
-
-Escape hatch for old login passwords.
-
-
-### `babelfishpg_tsql.escape_hatch_login_password_must_change`
-
-Escape hatch for login passwords `must_change` option.
-
-
-### `babelfishpg_tsql.escape_hatch_login_password_unlock`
-
-Escape hatch for login passwords unlock option.
-
-
-### `babelfishpg_tsql.escape_hatch_nocheck_add_constraint`
-
-Escape hatch for `WITH [NO]CHECK` option in `ALTER TABLE ADD`.
 
+### `babelfishpg_tsql.fmtonly`
 
-### `babelfishpg_tsql.escape_hatch_nocheck_existing_constraint`
+This parameter controls SQL Server compatibility related to the `FMTONLY` option.
 
-Escape hatch for `WITH [NO]CHECK` option in `ALTER TABLE` on existing
-constraint.
 
+### `babelfishpg_tsql.implicit_transactions`
 
-### `babelfishpg_tsql.escape_hatch_query_hints`
+This parameter enables implicit transactions.
+If no transaction is running, the server will start an implicit transaction
+for qualified commands when the `implicit_transactions` parameter is set to
+on.
 
-Escape hatch for query hints.
 
+### `babelfishpg_tsql.language`
 
-### `babelfishpg_tsql.escape_hatch_rowguidcol_column`
+This parameter controls the T-SQL compatibility language option.
```

Comment:
@rcv-aws @JimFinnerty @kuntalghosh @AvantikaDG 

I've dropped a note out requesting information about what babelfishpg_tsql.language option controls, how to set it, and what the valid values are... if any of you know, can you pass that along to me please or leave a note here?

Reply:
I've asked multiple times on multiple Slack channels and no one has answered - I've also grepped through the repos and can't find the parameter in the source code.  At this point, I'm going to assume that this is a leftover from early development, and remove it...

Reply:
Per our conversation, I've just added babelfishpg_tsql.language into the list of parameters that shouldn't be changed (in case it is available in an older/stable branch).